assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -329,6 +353,9 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
         setTransformStyle(target);
       }}
       onResizeEnd={({ target }) => {
+        if (isEditMode) {
+          setActionHappening(false);
```

Comment:
We already should have `setActionHappening` via either `useTransformHandler` or `isAnythingTransforming`.

Reply:
`isAythingTransforming` does not work as expected in this case since we actually need to update the element _before_ anything has started transforming. Will check if there's a way how to modify the logic to work for this case, too.